### PR TITLE
fix outdated hugegraph download link

### PR DESCRIPTION
**Pull request by liuxiaocs7**
as title

the original url got 404 error, replace it with url from [here](https://github.com/apache/incubator-hugegraph-toolchain/releases/tag/1.0.0)

### CODE REVIEW

**Review by imbajin — COMMENTED** [review]
Thanks a lot, I do remember there are **several outdated** download links like this. 

Could you help to search for them in full-text and fix them together? In addition, the doc of the **English** directory (current is `cn`) also needs to be corrected

**Comment by liuxiaocs7** [comment]
> Thanks a lot, I do remember there are **several outdated** download links like this.
> 
> Could you help to search for them in full-text and fix them together? In addition, the doc of the **English** directory (current is `cn`) also needs to be corrected
> 
> Also could add the basic instruction in hubble doc (visualize graph)
> 
> image

sure, i'm glad to do this!

now i deploy hugegraph and hubble by docker and try to run the [loader demo](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-loader/#4-%E5%AE%8C%E6%95%B4%E7%A4%BA%E4%BE%8B)

i'll add some basic instructions in hubble doc in next PR.

---

One question, When hubble creates a graph, we need to specify a `name` or `id`, but i don’t see the specified name or id in the [loader demo](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-loader/#4-%E5%AE%8C%E6%95%B4%E7%A4%BA%E4%BE%8B). How to ensure that the data is imported into the specified graph?

`schema.groovy` defines vertex/edges/properties/index and `struct.json` defines file format, etc.

image

**Comment by liuxiaocs7** [comment]
Also i noticed that [#447](https://github.com/apache/incubator-hugegraph-toolchain/pull/447) modify the default UI, should we wait for the docker environment to take effect before modifying the screenshot.

File: .github/workflows/validate-release.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       SCRIPT_PATH: hugegraph-dist/scripts/
-      URL_PREFIX: https://dist.apache.org/repos/dist/dev/incubator/hugegraph/
+      URL_PREFIX: https://downloads.apache.org/incubator/hugegraph/
```
Comment (liuxiaocs7) [inline .github/workflows/validate-release.yml:26]: not sure

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ sudo apt-get install wget -y
 brew install wget
 
 # 4. Download the hugegraph-svn directory (version number, pay attention to fill in the verification version, here we take 1.0.0 as an example)
-svn co https://dist.apache.org/repos/dist/dev/incubator/hugegraph/1.0.0/
+svn co https://downloads.apache.org/incubator/hugegraph/1.0.0/
```
Comment (liuxiaocs7) [inline content/en/docs/contribution-guidelines/validate-release.md:40]: not sure

File: dist/validate-release.sh [hunk 3]
```diff
@@ -21,7 +21,7 @@
 # 3. Compile the source package & run server & toolchain
 # 4. Run server & toolchain in binary package
 
-URL_PREFIX="https://dist.apache.org/repos/dist/dev/incubator/hugegraph/"
+URL_PREFIX="https://downloads.apache.org/incubator/hugegraph/"
```
Comment (liuxiaocs7) [inline dist/validate-release.sh:24]: not sure

**Comment by liuxiaocs7** [comment]
> > Thanks a lot, I do remember there are **several outdated** download links like this.
> > Could you help to search for them in full-text and fix them together? In addition, the doc of the **English** directory (current is `cn`) also needs to be corrected
> > Also could add the basic instruction in hubble doc (visualize graph)
> > image
> 
> sure, i'm glad to do this!
> 
> now i deploy hugegraph and hubble by docker and try to run the [loader demo](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-loader/#4-%E5%AE%8C%E6%95%B4%E7%A4%BA%E4%BE%8B)
> 
> i'll add some basic instructions in hubble doc in next PR.
> 
> One question, When hubble creates a graph, we need to specify a `name` or `id`, but i don’t see the specified name or id in the [loader demo](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-loader/#4-%E5%AE%8C%E6%95%B4%E7%A4%BA%E4%BE%8B). How to ensure that the data is imported into the specified graph?
> 
> `schema.groovy` defines vertex/edges/properties/index and `struct.json` as mapping file defines file format, etc.
> 
> image

find here:

image

image

File: .github/workflows/validate-release.yml (hunk 1 above)
Comment (imbajin) [reply .github/workflows/validate-release.yml:26]: yep, **no need** to update it. 
indeed, the `dist.apache.org` is a **svn url** 🔁 (stage the package before release)

File: content/en/docs/contribution-guidelines/validate-release.md (hunk 2 above)
Comment (imbajin) [reply content/en/docs/contribution-guidelines/validate-release.md:40]: same, note `svn` prefix

File: dist/validate-release.sh (hunk 3 above)
Comment (imbajin) [reply dist/validate-release.sh:24]: same

**Comment by liuxiaocs7** [comment]
seem works:

server ok
```
PS C:\Users\liuxiaocs> curl 192.168.34.164:18080


StatusCode        : 200
StatusDescription : OK
Content           : {"service":"hugegraph","version":"1.0.0","doc":"https://hugegraph.apache.org/docs/","api_doc":"https://hugegraph.apache.org/docs/clients/","apis":["auth","cyp
                    her","filter","graph","gremlin","job","met...
RawContent        : HTTP/1.1 200 OK
                    Content-Length: 270
                    Content-Type: application/json

                    {"service":"hugegraph","version":"1.0.0","doc":"https://hugegraph.apache.org/docs/","api_doc":"https://hugegraph.apache.org/docs...
Forms             : {}
Headers           : {[Content-Length, 270], [Content-Type, application/json]}
Images            : {}
InputFields       : {}
Links             : {}
ParsedHtml        : mshtml.HTMLDocumentClass
RawContentLength  : 270
```

create graph

image

insert data

```
bash bin/hugegraph-loader.sh -g hugegraph -p 18080 -f example/file/struct.json -s example/file/schema.groovy
```

```
[liuxiao@dev apache-hugegraph-loader-incubating-1.0.0]$ bash bin/hugegraph-loader.sh -g hugegraph -p 18080 -f example/file/struct.json -s example/file/schema.groovy
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/liuxiao/Code/hugegraph/apache-hugegraph-toolchain-incubating-1.0.0/apache-hugegraph-loader-incubating-1.0.0/lib/log4j-slf4j-impl-2.18.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/liuxiao/Code/hugegraph/apache-hugegraph-toolchain-incubating-1.0.0/apache-hugegraph-loader-incubating-1.0.0/lib/slf4j-log4j12-1.6.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
>> HugeGraphLoader worked in NORMAL MODE
vertices/edges loaded this time : 8/6
--------------------------------------------------
count metrics
    input read success            : 14
    input read failure            : 0
    vertex parse success          : 8
    vertex parse failure          : 0
    vertex insert success         : 8
    vertex insert failure         : 0
    edge parse success            : 6
    edge parse failure            : 0
    edge insert success           : 6
    edge insert failure           : 0
--------------------------------------------------
meter metrics
    total time                    : 0.131s
    read time                     : 0.04s
    load time                     : 0.091s
    vertex load time              : 0.069s
    vertex load rate(vertices/s)  : 115
    edge load time                : 0.065s
    edge load rate(edges/s)       : 92
```

**Comment by imbajin** [comment]
> One question, When hubble creates a graph, we need to specify a name or id, but i don’t see the specified name or id in the [loader demo](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-loader/#4-%E5%AE%8C%E6%95%B4%E7%A4%BA%E4%BE%8B). How to ensure that the data is imported into the specified graph?

schema.groovy defines vertex/edges/properties/index and struct.json as mapping file defines file format, etc.

Current **loader doc** is also **too long** to read (users may get lost in the page), it's not friendly and the doc should separate into multi subpages. (show the most important info for user first)   [TODO]

> Graph ID?

GraphID & GraphName is really confused for new users to set the proper value, also need enhance the description or doc [TODO] 

> should we wait for the docker environment to take effect before modifying the screenshot.?

No need, the docker images could update anytime (currently it build once per week, we hope there is a way to autobuild it by PR merged or like a **dockerhub hook**) 

> i'll add some basic instructions in hubble doc in next PR.

fine

**Comment by liuxiaocs7** [comment]
`g.V()` shows results that matchs, thanks!

image

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 4]
```diff
@@ -38,9 +38,9 @@ sudo apt-get install wget -y
 brew install wget
 
 # 4. 下载 hugegraph-svn 目录 (版本号注意填写此次验证版本, 这里以1.0.0为例)
-svn co https://dist.apache.org/repos/dist/dev/incubator/hugegraph/1.0.0/
+svn co https://downloads.apache.org/incubator/hugegraph/1.0.0/
```
[the remark this answers is not in the document]
Comment (imbajin) [reply content/cn/docs/contribution-guidelines/validate-release.md:41]: ignore it in all `validate-release` doc or script 💯

**Comment by imbajin** [comment]
> `g.V()` shows results that matchs, thanks!

And we should also add a **groovy script** in hubble's doc so that users could add a graph with `one-click` paste

Just like the groovy example in [server](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-dist/src/assembly/static/scripts/example.groovy)

File: .github/workflows/validate-release.yml (hunk 1 above)
Comment (liuxiaocs7) [reply .github/workflows/validate-release.yml]: got it, thanks!

**Comment by imbajin** [comment]
also need revoke the url in `validate-release` doc (with svn url)

**Comment by liuxiaocs7** [comment]
> also need revoke the url in `validate-release` doc (with svn url) [meet some network problems. so can't review or modify your code directly...]
> 
> image

got, fixed!

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ brew install wget
 # 4. Download the hugegraph-svn directory (version number, pay attention to fill in the verification version, here we take 1.0.0 as an example)
 svn co https://dist.apache.org/repos/dist/dev/incubator/hugegraph/1.0.0/
 # (Note) If svn downloads a file very slowly, you can consider wget to download a single file, as follows (or consider using a proxy)
-wget https://dist.apache.org/repos/dist/dev/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0.tar.gz
+wget https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0.tar.gz
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:42]: ditto

**Review by imbajin — APPROVED** [review]

**Pull request merged**